src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -31,6 +31,11 @@ class Tribe__Events__Linked_Posts__Chooser_Meta_Box {
 	 */
 	protected $singular_name;
 
+	/**
+	 * @var string
+	 */
+	 protected $singular_name_lowercase;
```

Comment:
```suggestion
	protected $singular_name_lowercase;
```
That's my fault, sorry